terraform/organizations.tf

```diff
@@ -16,6 +18,7 @@ resource "aws_organizations_organization" "default" {
     "ram.amazonaws.com",
     "reporting.trustedadvisor.amazonaws.com",
     "securityhub.amazonaws.com",
+    "ssm.amazonaws.com",
```

Comment:
This addition brings the resource up to date (SSM was configured via click-ops).